src/backend/services/MonitoringService.ts

```
@@ -0,0 +1,379 @@
+      total_requests: this.stats.requests.count,
+      avg_response_time: Math.round(avgResponseTime),
+      error_rate: Math.round(errorRate * 100) / 100,
+      gemini_calls: this.stats.geminiCalls,
+      pdf_processed: this.stats.pdfProcessed,
+      analysis_completed: this.stats.analysisCompleted,
+      active_sessions: this.stats.activeSessions.size
+    };
+  }
+
+  // Get recent logs
+  async getRecentLogs(limit: number = 100): Promise<LogEntry[]> {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)